src/styles/blocks/stop-watch.scss

```
@@ -0,0 +1,65 @@
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stop-watch {
+  position: relative;
+
+  width: $watch-size;
+  height: $watch-size;
+  border: 4px dotted black;
+  border-radius: 50%;
+
+  &__dot {
+    z-index: 1;
+    position: absolute;
+    top: $dot-center;
+    left: $dot-center;
+
+    height: $dot-size;
+    width: $dot-size;
+    border-radius: 50%;
+
+    background-color: rgb(58, 162, 180);
+  }
+
+  @keyframes clock-hand-rotate {
+    from {
+        transform: rotate(0deg)
+    }
+    to {
+        transform: rotate(360deg)
+    }
+  }
+
+  &__big-hand {
+    position: absolute;
+    top: 0;
+    right: $watch-size / 2 - $big-hand-width / 2;
```

you can make variables for this
```suggestion
    right: $half-watch-size -$same..;
```